sources.json

```diff
@@ -581,5 +581,19 @@
     "type": "command",
     "tags": ["development", "tooling", "testing", "browser"],
     "license": "mit"
+  },
+    {
+    "owner": "SAP",
+    "repo": "ui5-tooling-extensions",
+    "subpath": "packages",
+    "subpackages": [
+      {
+        "name": "@ui5/middleware-code-coverage",
+        "addedToBoUI5": "2023-03-22T14:12:29.389Z",
```

Comment:
if you ready to merge, just update this

Reply:
I've set it to next week wed. When release of 1.113 is out, i'll ping you here.